City-of-Pori/pori-kada
# PORI-308 Enabled revisions for news item, page, landing page and attr…

…action

To test:

1. drush fra -y && drush cc all
2. Go to news item, page, landing page and attraction card and make sure revisions are on and new one is created everytime node is edited.
3. Log in with content editor or web journalist and make sure you can't remove revisions but you are able to revert to them.
4. Log in with admin and make sure you are able to delete revisions.
